packages/live-status-gateway/src/collections/partInstances.ts

```diff
@@ -0,0 +1,117 @@
+import { Logger } from 'winston'
+import { CoreHandler } from '../coreHandler'
+import { CollectionBase, Collection, CollectionObserver } from '../wsHandler'
+import { CoreConnection } from '@sofie-automation/server-core-integration'
+import { DBRundownPlaylist } from '@sofie-automation/corelib/dist/dataModel/RundownPlaylist'
+import { DBPartInstance } from '@sofie-automation/corelib/dist/dataModel/PartInstance'
+import { unprotectString } from '@sofie-automation/corelib/dist/protectedString'
+
+export enum PartInstanceName {
+	cur = 'cur',
```

Comment:
If this is supposed to be `current` then please call it `current`. 
I don't think you mean an actual Cur
https://en.wikipedia.org/wiki/Cur

Reply:
Renamed in https://github.com/nrkno/sofie-core/pull/777/commits/ce0911fd2b9a01f267c2d186da698ac687c5dcd6